app/uk/gov/hmrc/helptosavefrontend/views/register/check_your_details.scala.html

```diff
@@ -20,5 +20,100 @@
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
 @import uk.gov.hmrc.helptosavefrontend.views.helpers.errorPrefix
 @import uk.gov.hmrc.helptosavefrontend.controllers.routes
+@import uk.gov.hmrc.helptosavefrontend.models.userinfo.UserInfo
+@import cats.syntax.show._
+@import uk.gov.hmrc.helptosavefrontend.models.userinfo.UserInfo.localDateShow
 
-Check your details
\ No newline at end of file
+@(userInfo: UserInfo, email: String, bankDetails: BankDetails)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(
+title = Messages("hts.global.title", Messages("hts.register.you-are-eligible.title.h1"), "", Messages("hts.global.title-suffix")),
+sidebarLinks = None,
+bodyClasses = None) {
+
+<a onclick="goBack();return false;">Back</a>
```

Comment:
You can factor the back button stuff out into a helper view to avoid copying and pasting the same code

Reply:
its all temporary stuff, but i can do it